src/palace/manager/integration/license/opds/odl/importer.py

```diff
@@ -17,7 +17,7 @@
 from palace.manager.integration.license.opds.opds2.settings import OPDS2ImporterSettings
 from palace.manager.integration.license.opds.requests import (
     BaseOpdsHttpRequest,
-    OPDS2AuthType,
+    OpdsAuthType,
```

Comment:
I renamed this enum from `OPDS2AuthType` to `OpdsAuthType`, since its not OPDS2 specific and its being using in non-OPDS2 code. This could come out to a separate PR if needed.